pkg/client/clientset_generated/release_1_5/typed/core/v1/fake/fake_pod_expansion.go

```diff
@@ -44,3 +45,14 @@ func (c *FakePods) GetLogs(name string, opts *v1.PodLogOptions) *restclient.Requ
 	_, _ = c.Fake.Invokes(action, &v1.Pod{})
 	return &restclient.Request{}
 }
+
+func (c *FakePods) Evict(eviction *policy.Eviction) error {
```

Comment:
This is to fix the unit test failure. The function is required by the PodInterface. @nikhiljindal